Makefile

```
@@ -1,6 +1,6 @@
 .PHONY: run-explorer run-tests run-linters build-ui build-python build-docker run-docker compose-up
 
-version="0.31.1"
+version="0.40.0"
```

Why skipping 9 versions?
```suggestion
version="0.32.0"
```